src/persist-client/src/internal/state.rs

```diff
@@ -1466,6 +1408,14 @@ pub(crate) mod tests {
 
     use super::*;
 
+    const LEASE_DURATION_MS: u64 = 900 * 1000;
+    fn debug_state() -> HandleDebugState {
+        HandleDebugState {
+            hostname: "debug".to_owned(),
+            purpose: "finding the bugs".to_owned(),
+        }
+    }
```

Comment:
Covered by unit tests but not by tests of higher levels. More as an information, it does not look too critical. Feel free to ignore.